src/docker.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, chain_name, docker_tag):
 
     def extract_resources_from_docker(self):
         if self.chain_name in ['spiritnet', 'peregrine', 'peregrine-stg-kilt']:
```

Comment:
@jonathanudd this change affects other deploys as well such as spiritnet. But we should already be using the spec url config param everywhere right?

Reply:
:+1: